tests/test_serial_to_zarr.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
```

Comment:
Alternatively, the tests directory could be inside the serial_convert directory (having a `src` and `tests` structure). Just a thought.